docs/releasenotes/index.rst

```diff
@@ -10,6 +10,14 @@ You can learn more about where we might be going with future releases by:
 * Stopping by the ``#clusterhq`` channel on ``irc.freenode.net``.
 * Visiting our GitHub repository at https://github.com/ClusterHQ/flocker.
 
+v1.3
+====
+
+* Flocker API has new endpoints to create, list, renew, and delete lease on a dataset.
+* Flocker now works on servers where ``SELinux`` is enabled.
+* Flocker is more accurate in figuring out attached block device path on ``Cinder`` backend.
+* Flocker now has smaller log footprint from ``EBS`` storage driver on success path.
```

Comment:
These sentences don't need to be shortened - I suggest something like this to make it clearer:

> * Flocker is more accurate in figuring out the attached block device path on a ``Cinder`` backend.
> * Flocker now has a smaller log footprint from the ``EBS`` storage driver on the success path.